README.md

```diff
@@ -1751,7 +1747,7 @@ Setting config preset as default for aspera
 saving config file
 Done.
 You can test with:
-$ ascli aspera user info show
+$ ascli oncloud user info show
```

Comment:
I am ok with `oncloud` but don't we use `aoc` everywhere else in the company to refer to Aspera on Cloud?

Reply:
We'll potentially address this in a separate PR.

Reply:
yes, see my comment to david in another there here below.